tests/formatter_tests.js

```diff
@@ -80,6 +83,134 @@ VF.Test.Formatter = (function() {
+      var stave11 = vf.Stave({ y: 20, width: width + 20 });
+      var stave21 = vf.Stave({ y: 130, width: width + 20 });
+      formatter.format([voice11, voice21], width);
+      stave11.setContext(ctx).draw();
+      stave21.setContext(ctx).draw();
+      voice11.draw(ctx, stave11);
+      voice21.draw(ctx, stave21);
+      beams21.forEach(function(b) {
+        b.setContext(ctx).draw();
+      });
+      beams11.forEach(function(b) {
+        b.setContext(ctx).draw();
+      });
+      if (voice11.tickables[1].getX() >= voice21.tickables[1].getX()) {
+        console.warn('Second note of voice 1 is not to the right of the second note of voice 2');
+      }
+      ok(true);
+    },
+    unalignedNoteDurations2: function(options) {
+      var notes1 = [
+        new VF.StaveNote({ keys: ['b/4'], duration: '8r' }),
+        new VF.StaveNote({ keys: ['g/4'], duration: '16' }),
+        new VF.StaveNote({ keys: ['c/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['e/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['g/4'], duration: '16' }),
+        new VF.StaveNote({ keys: ['c/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['e/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['b/4'], duration: '8r' }),
+        new VF.StaveNote({ keys: ['g/4'], duration: '16' }),
+        new VF.StaveNote({ keys: ['c/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['e/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['g/4'], duration: '16' }),
+        new VF.StaveNote({ keys: ['c/5'], duration: '16' }),
+        new VF.StaveNote({ keys: ['e/5'], duration: '16' }),
+      ];
+      var notes2 = [
+        new VF.StaveNote({ keys: ['a/4'], duration: '16r' }),
+        new VF.StaveNote({ keys: ['e/4.'], duration: '8d' }),
+        new VF.StaveNote({ keys: ['e/4'], duration: '4' }),
+        new VF.StaveNote({ keys: ['a/4'], duration: '16r' }),
+        new VF.StaveNote({ keys: ['e/4.'], duration: '8d' }),
+        new VF.StaveNote({ keys: ['e/4'], duration: '4' }),
+      ];
+      var vf = VF.Test.makeFactory(options, 750, 280);
+      const context = vf.getContext();
+      var voice1 = new VF.Voice({ num_beats: 4,  beat_value: 4 });
+      voice1.addTickables(notes1);
+      var voice2 = new VF.Voice({ num_beats: 4,  beat_value: 4 });
+      voice2.addTickables(notes2);
+      var formatter = new VF.Formatter();
+      formatter.joinVoices([voice1]);
+      formatter.joinVoices([voice2]);
+      var width = formatter.preCalculateMinTotalWidth([voice1, voice2]);
+      formatter.format([voice1, voice2], width + 20);
+      var stave1 = new VF.Stave(10, 40, width + 30);
+      var stave2 = new VF.Stave(10, 100, width + 30);
+      stave1.setContext(context).draw();
+      stave2.setContext(context).draw();
+      voice1.draw(context, stave1);
+      voice2.draw(context, stave2);
+      if (voice1.tickables[1].getX() >= voice2.tickables[1].getX()) {
```

Comment:
@AaronDavidNewman here the second note of voice1 should also be to the right of second note of voice2. In the if condition checking the failure, it should be, I believe, <= and not >=.
```js
if (voice1.tickables[1].getX() <= voice2.tickables[1].getX())
```
depending on the answer from @0xfe prehaps:
```js
if (voice1.tickables[1].getX() < (voice2.tickables[1].getX() + voice2.tickables[1].getWidth()))
```

Reply:
This is done.  I added the '=' because the X locations are currently the same (also why I didn't catch <> issue).